src/prebid.js

```diff
@@ -514,7 +504,9 @@ $$PREBID_GLOBAL$$.requestBids = function ({ bidsBackHandler, timeout, adUnits, a
   }
 
   //set timeout for all bids
-  setTimeout(bidmanager.executeCallback, cbTimeout);
+  const timedOut = true;
+  const timeoutCallback = bidmanager.executeCallback.bind(bidmanager, timedOut);
+  setTimeout(timeoutCallback, cbTimeout);
```

Comment:
We have opted to base the timeout not on Javascript (which can be unreliable) but on a comparison of a bid's `timeToResponse` and the request bids timeout as shown here:
https://github.com/prebid/Prebid.js/pull/500/files#diff-96cd0debf7b766d72d2bcf987385e960L124